#! /bin/sh /usr/share/dpatch/dpatch-run
## gpsd_dbus.h-buildfix.dpatch by Bernd Zeimetz <bzed@debian.org>
##
## DP: r4543 | esr | 2007-12-12 22:34:19 +0100 (Wed, 12 Dec 2007) | 2 lines
## DP: Renable building with DBUS.

@DPATCH@
diff -urNad gpsd~/gpsd_dbus.h gpsd/gpsd_dbus.h
--- gpsd~/gpsd_dbus.h	2007-06-01 23:17:26.000000000 +0200
+++ gpsd/gpsd_dbus.h	2007-12-15 02:13:51.000000000 +0100
@@ -6,6 +6,8 @@
 
 #include <dbus/dbus.h>
 
+#include "gpsd.h"
+
 int initialize_dbus_connection (void);
 void send_dbus_fix (struct gps_device_t* channel);
 
